### ISSUE TITLE
Add feature extractors with less parameters

### ISSUE THREAD

**Issue by avoroshilov** [avoroshilov/neural-style#1]
Based on this [TF-slim models page](https://github.com/tensorflow/models/tree/master/slim#pre-trained-models), good candidate could be `Inception V2`, good accuracy and small weight of pre-trained model (40 MBs).
Another candidate could be [SqueezeNet](https://arxiv.org/abs/1602.07360).

**Comment by avoroshilov** [comment]
Added SqueezeNet v1.1 backend in https://github.com/avoroshilov/neural-style/commit/609b7716079979fd1977a901f54bd35ed58e3a1e
